test/scripts/e2e_basic_start_stop.sh

```diff
@@ -55,17 +55,18 @@ verify_none_running
 #----------------------
 # Test that we can start & stop a generic node with no overrides
 echo Verifying a generic node will start using goal
-${BINDIR}/goal node start -d ${DATADIR}
+goal node start -d ${DATADIR}
 verify_at_least_one_running
 
 echo Verifying we can stop it using goal
-${BINDIR}/goal node stop -d ${DATADIR}
+goal node stop -d ${DATADIR}
 verify_none_running
 
 #----------------------
 # Test that we can start a generic node straight with no overrides
 echo Verifying a generic node will start directly
-${BINDIR}/algod -d ${DATADIR} &
+algod -d ${DATADIR} &
+sleep 1
```

Comment:
If this does becomes an issue, this is not the correct place to fix it. you might want to try to sleep in case the pgrep  returns no pids in update_running_count

Reply:
I'm surprised the answer isn't to make verify_at_least_one_running have a retry loop like verify_one_running has

Reply:
I just updated verify_at_least_one_running to include the retry loop and removed the sleep statements. Good thinking.